src/armor_layers.cpp

```diff
@@ -175,12 +175,15 @@ void player::sort_armor()
             } else {
                 mvwprintz(w_sort_middle, cont_h - 12 + i, 2, c_ltgray, "%s:", armor_cat[i].c_str());
             }
-            mvwprintz(w_sort_middle, cont_h - 12 + i, middle_w - 16, c_ltgray, "%d+%d = ", armorenc,
-                      enc - armorenc);
+            true_enc = enc - armorenc;
+            char my_spaces[]    = "   ";    // lol
+            char *spaces        = (char*)&my_spaces;
```

Comment:
`&my_spaces` is a `char**`, not a `char*` not at all. Btw. why not `const char*` and `const char my_spaces[]`.

This example complies fine for me:
```C++
    static const char my_space[] = "   ";
    const char *x = my_space;
    x++;
```

And C++ is *not* weird. Humans are weird, they say "or", but they mean "xor".